### DISCUSSION TITLE
Successfully Ran Getting Started on Windows

### DISCUSSION THREAD

**Discussion by Steve-O-Murphy** [flyteorg/flyte#1767] [General]
Hey Flyte team,

Just FYI, I successfully ran the '[1. Implement](https://docs.flyte.org/en/latest/getting_started.html)' step on Windows (10). Wanted to let you know because you had asked for feedback.

**Comment by EngHabu** [comment]
This is awesome! thank you for letting us know. Do you have any questions or feedback about it?

**Comment by cgrisham** [comment]
I have not been able to run the Flyte getting started on windows. When I try to execute the workflow in the demo cluster I get this error 

"[1/1] currentAttempt done. Last Error: USER::Pod failed. No message received from kubernetes.
[feb1d6fc03e6c4a87aa0-n0-0] terminated with exit code (137). Reason [OOMKilled]. Message: 
tar: Removing leading `/' from member names"

**Reply by kumare3** [reply]
@cgrisham this is because the emulated version of docker are consuming more memory for some reason in the getting started. We are just releasing a fix, but until then if you can update the getting started code as follows

```
@task(limits=Resources(mem="500Mi"))
def generate_normal_df(n: int, mean: float, sigma: float) -> pd.DataFrame:
    return pd.DataFrame({"numbers": np.random.normal(mean, sigma, size=n)})


@task(limits=Resources(mem="500Mi"))
def compute_stats(df: pd.DataFrame) -> typing.Tuple[float, float]:
    return float(df["numbers"].mean()), float(df["numbers"].std())
```
As done on this issue that would be great - https://github.com/flyteorg/flyte/issues/2376

**Comment by katrogan** [comment]
hi @cgrisham alternatively, our latest Flyte beta release should also work if you don't want to muck with task resources

```
flytectl demo start --pre
```

and this should have increased resources for all tasks applied by default.

**Comment by cgrisham** [comment]
@kumare3  @katrogan Thank you both so much for your responsiveness. Both of these methods work to resolve the issue !
